proto/path/protocol/morse.proto

```diff
@@ -32,7 +32,11 @@ enum MorseEndpointErrorType {
   // This error indicates the endpoint returned an HTTP response with a mismatch between ContentLength header and actual body length.
   // Mostly encountered as ContentLength > 0 with an actual body length of 0.
   MORSE_ENDPOINT_ERROR_HTTP_LENGTH_HEADER_MISMATCH = 10;
+
   MORSE_ENDPOINT_ERROR_INTERNAL = 11;  // Added for internal gateway errors
+
+  // This error indicates the endpoint returned an error on executing the HTTP request.
+  MORSE_ENDPOINT_ERROR_EXECUTING_HTTP_REQUEST = 12;
```

Comment:
Why don't we use one of the 4xx or 5xx responses? E.g. enum 8 or 9

Reply:
This is done to differentiate the type of errors returned by endpoints/SDK in observations/data.

Reply:
No need for a change, but can you #PUC the expectation here?

Are we 100% sure this implies NOT a 4xx and NOT a 5xx?

Just looking for clarity.